Jellyfin.Plugin.Tvdb/TvdbClientManager.cs

```diff
@@ -542,7 +542,18 @@ public async Task<IReadOnlyList<ArtworkType>> GetArtworkTypeAsync(CancellationTo
         var updatesClient = _serviceProvider.GetRequiredService<IUpdatesClient>();
         await LoginAsync().ConfigureAwait(false);
         var updatesResult = await updatesClient.UpdatesAsync(since: fromTime, type: type, action: action, cancellationToken: cancellationToken).ConfigureAwait(false);
-        return updatesResult.Data;
+        IEnumerable<EntityUpdate> updates = updatesResult.Data;
```

Comment:
I guess you could just use var here instead of the full type?

Reply:
updatesResult.Data is a IReadonlylist. Need to cast to IEnumerable so I can concat the subsequent updatesResult.Data.

Reply:
You should build a list instead of concat.